pkg/square/square.go

```diff
@@ -200,7 +188,7 @@ func (s Square) Size() int {
 // avoid breaking the api. In future versions there will not be a copy of this
 // code here.
 func Size(len int) int {
```

Comment:
[blocking] the godoc here needs to be updated to remove this portion:

"This function is currently a wrapper around the da packages equivalent function to avoid breaking the api."

Sorry can't provide a suggestion because lines not modified.